x-pack/platform/plugins/shared/fleet/server/saved_objects/index.ts

```diff
@@ -1137,6 +1138,16 @@ export const getSavedObjectTypes = (
           proxy_id: { type: 'keyword' },
         },
       },
+      modelVersions: {
+        '1': {
+          changes: [
+            {
+              type: 'mappings_addition',
+              addedMappings: {},
```

Comment:
explanation of this field can be found in a previous PR: https://github.com/elastic/kibana/pull/208091/files#r1971649339